source/manual/dns.html.md

```diff
@@ -42,23 +42,44 @@ We use a Jenkins job that publishes changes to `publishing.service.gov.uk`. The
 job uses [Terraform](https://www.terraform.io/) and pushes changes to the
 selected provider.
 
-### Jenkins
+### Deployment
 
-When the changes have been reviewed and merged, you can deploy them using [the
-"Deploy DNS" Jenkins job](https://deploy.publishing.service.gov.uk/job/Deploy_DNS/).
+**Always `plan` first, check that the output is what you expect, then `apply`.**
 
-You will need to [assume the appropriate role](/manual/access-aws-console.html)
-and copy and paste the credentials in to the Jenkins job.
+When the changes have been reviewed and merged, you can deploy them in your terminal
+by using at least version `v2.15.0` of [`gds-cli`](/manual/access-aws-console.html).
```

Comment:
Same point as https://github.com/alphagov/govuk-developer-docs/pull/2590#discussion_r436521657 - the link now redirects, so would be worth updating, but also a more general point that `gds-cli` as link text sounds like it will take you to the repo.